docs/painless/painless-guide/painless-datetime.asciidoc

```diff
@@ -600,18 +600,18 @@ if (doc.containsKey('start') && doc.containsKey('end')) { <1>
 contain a specific field. Use the `containsKey` method call on the `doc` input
 to ensure a field exists as part of the index for the current document.
 <2> Some fields within a document may have no values. Use the `size` method
-call on a field within the `doc` input to ensure that field has at least one
+call on a field within the `doc` input to ensure that a field has at least one
```

Comment:
I believe the previous change was a bit more clear.

`that field` refers to the field on which the `size` method was called. Here's my recommendation:

> call on a field within the `doc` input to ensure the field has at least one